src/test/resources/org/prebid/server/it/amp/test-appnexus-bid-request.json

```diff
@@ -70,16 +70,6 @@
             }
           ]
         },
-        "currency": {
```

Comment:
Is is strange that in this method currency just disappeared (not migrate to the new place)
 
Please change `test-amp-stored-request.json` (currency rates place)  and fix this test.